Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:6.7.0-jdk14 as builder
```

Comment:
As the build.gradle suggests, the java version should be 17

Reply:
Nope, it suggest java 8 but it is running on java 14 since the version 2 [https://github.com/GravenDev/PrivateRooms/blob/1a4fc3b835aa8eca640105692bf45c0be7c0d22a/build.gradle#L42](url)